styles/blocks/category.scss

```
@@ -0,0 +1,16 @@
+@use '../utils/mixins.scss' as *;
+
+.category {
+  &__name {
+    color: $main-font-color;
+    font-size: 16px;
+    font-family: Inter;
```

Instead of specifying font-family on every element you can have a global font styles you can set on body element.
Also it is a good practice to add fallback fonts which will be apllied when Inter font is not accessible
```suggestion
    font-family: Inter, 'sans-serif';
```